src/uwtools/utils/cli_helpers.py

```diff
@@ -32,15 +33,15 @@ def get_file_type(arg): #pylint: disable=unused-variable
     logging.critical(msg)
     raise ValueError(msg)
 
-def path_if_file_exists(arg): #pylint: disable=unused-variable
+def path_if_file_exists(arg):
     ''' Checks whether a file exists, and returns the path if it does. '''
     if not os.path.exists(arg):
         msg = f'{arg} does not exist!'
         raise argparse.ArgumentTypeError(msg)
 
     return os.path.abspath(arg)
 
-def setup_logging(user_args, log_name=None): #pylint: disable=unused-variable
+def setup_logging(user_args, log_name=None):
```

Comment:
Please keep these as they are. Applying the disable method to the entire file misses other instances outside these handful of lines that could be important to know in the linting process.